### PR TITLE
Fixes #33621 - Remove experimental host wizard

### PR DESCRIPTION
**Pull request by Ron-Lavi**
The page was never created actually.
it was more of a test for our React router and a future base for a future host wizard :)

### CODE REVIEW

**Comment by Ron-Lavi** [comment]
cc @tbrisker @amirfefer

File: app/registries/menu/loader.rb [hunk 1]
```diff
@@ -94,12 +94,7 @@ def self.load
       end
 
       Manager.map :labs_menu do |menu|
```
Comment (tbrisker) [inline app/registries/menu/loader.rb:96]: Do we still need this empty menu? Or is it used by some plugins?

Comment (tbrisker) [reply app/registries/menu/loader.rb:96]: never mind. looks like it is.

Comment (Ron-Lavi) [reply app/registries/menu/loader.rb:96]: it is used by REX for the new job wizard and also by Katello CV if I recall

**Review by tbrisker — APPROVED** [review]
Thanks @laviro ! LGTM

**Comment by ezr-ondrej** [comment]
I guess there is no point of cherry-picking into the 3.0 now, is there?
@Ron-Lavi could you open a PR if you think it is still relevant?

**Pull request merged**